documentation/tutorials/auth0-quickstart.md

```diff
@@ -117,7 +117,7 @@ defmodule MyApp.Accounts.User do
       argument :user_info, :map, allow_nil?: false
       argument :oauth_tokens, :map, allow_nil?: false
       upsert? true
-      upsert_identity :email
+      upsert_identity :unique_email
```

Comment:
This is the atom people will have configured if they follow the guide from here https://hexdocs.pm/ash_authentication_phoenix/getting-started-with-ash-authentication-phoenix.html